downloads/is_installation.adoc

```diff
@@ -0,0 +1,76 @@
+= Installation
+:page-layout: project
+
+== If you already have JBDSIS 8.1.0 installed...
+
+Simply start jbdevstudio or eclipse-with-jbds, then:
```

Comment:
this is all very version specific. lets update it to be bit more useful for future updates.